.github/workflows/build_all.yml

```diff
@@ -3,32 +3,32 @@ name: Build all
 on:
   push:
     branches:
-     - main
-     - release/*
+      - main
+      - release/*
     paths:
-     - 'deps/**'
-     - 'src/**'
-     - '**/CMakeLists.txt'
-     - 'version.inc'
-     - 'localization/**'
-     - 'resources/**'
-     - ".github/workflows/build_*.yml"
+      - 'deps/**'
+      - 'src/**'
+      - '**/CMakeLists.txt'
+      - 'version.inc'
+      - 'localization/**'
+      - 'resources/**'
+      - ".github/workflows/build_*.yml"
 
   pull_request:
     branches:
-     - main
-     - release/*
+      - main
+      - release/*
     paths:
-     - 'deps/**'
-     - 'src/**'
-     - '**/CMakeLists.txt'
-     - 'version.inc'
-     - ".github/workflows/build_*.yml"
-     - 'BuildLinux.sh'
-     - 'build_release_vs2022.bat'
-     - 'build_release_macos.sh'
-
-  workflow_dispatch: # allows for manual dispatch
+      - 'deps/**'
+      - 'src/**'
+      - '**/CMakeLists.txt'
+      - 'version.inc'
+      - ".github/workflows/build_*.yml"
+      - 'BuildLinux.sh'
+      - 'build_release_vs2022.bat'
+      - 'build_release_macos.sh'
+
+  workflow_dispatch:

```

Comment:
I guess you ran a code formatter against these files which does not preserve comments.
Generally speaking, try avoid doing unnecessary changes to the file, especially formatting the entire file. It's usually acceptable if you only reformat the part that you changed. Otherwise the PR will be hard to review with all theses unrelated destractions, and also this could create unnecessary conflictions to others.